client/rpc/src/main/kotlin/net/corda/client/rpc/internal/RPCClientProxyHandler.kt

```
@@ -142,6 +143,19 @@ internal class RPCClientProxyHandler(
                 }
             }
         }
+
+        @Suppress("TooGenericExceptionCaught")
+        private fun closeObservable(observable: UnicastSubject<Notification<*>>) {
+            // Notify listeners of the observables that the connection is being terminated.
+            try {
+                observable.onError(ConnectionFailureException())
+            } catch (ex: OnErrorNotImplementedException) {
+                // Indicates the observer does not have any error handling.
+                log.warn("Closed connection on observable $observable whose observers have no error handling.")
```

what is the `$observable` part going to print - will this contain something that the user can use for correlation purposes (e.g. observable ID)?

---

The warning looks like:
`[WARN] 09:20:50,884 [Time-limited test] internal.RPCClientProxyHandler. - Closed connection on observable rx.subjects.UnicastSubject@49b6bb49 whose observers have no error handling.`

- This is unhelpful
- It's really hard to get the subscriber from the subscription (we would need to change rx code).
- It looks like subscribing without error handling is a valid use case that we have a lot in the code.

I think we should ignore this exception altogether.

---

Dropped it down to `trace` level just in case, and removed the `observable` print